### ISSUE TITLE
Please remove IP from blacklist

### ISSUE THREAD

**Issue by isaguimiot** [docker/hub-feedback#1764]
Hi, 

This is an emergency request! We found out that we've been blacklisted by Docker Hub: we're using concourse CI with thousands of pipelines running every day. 

Here are the 2 IPs that seem to be blocked by Docker hub:

- 40.86.228.178
- 40.86.223.59

We will obviously try to deploy a local registry mirroring Docker Hub, to avoid massive direct pulling on your servers. But meanwhile, we need to have access to Docker Hub, as we have hundreds of users that absolutely need Concourse ASAP. Could you please remove us from the blacklist?

if you need to contact me, please let me know how to proceed.

Thanks a lot for your understanding,

Isabelle

**Comment by meticulous-dft** [comment]
Removed these 2 ips.

**Comment by isaguimiot** [comment]
Hi!

Thanks for your quick answer. Unfortunately, I still get a 500 error when running a curl from my VM 40.86.223.59 :

```
> curl https://hub.docker.com
<html><body><h1>500 Server Error</h1>
An internal server error occured.
</body></html>
```

Is there anything I need to do on my side ? Or do I have to wait for some restart on your side ?

Thanks!

**Comment by cowsrule** [comment]
I double checked our end, and it does not look like we are actively blocking your IPs. Additionally I verified that we are seeing traffic from that address that is not being blocked.

**Comment by isaguimiot** [comment]
Thank you, now it works on our side! I suppose there was a delay after the removal.
We plan to deploy a local registry in the next few weeks, thank you for your patience.

**Comment by yenernoyaner** [comment]
Hi there @cowsrule, is there a list of blocked ips to see if our ip addresses are on the list. 
We are having trouble to connet hub.docker.com either. 
Thanks in advance.

**Comment by cowsrule** [comment]
There is not. If you are receiving 429s/500s please open a new issue and include your IP address. Thanks!
